### ISSUE TITLE
Travis-ci make verify - signal killed

### ISSUE THREAD

**Issue by coreydaley** [openshift/origin#13969]
https://travis-ci.org/openshift/origin/builds/227274282
```
 PROTO_OPTIONAL=1 make verify
hack/build-go.sh  
++ Building go targets for linux/amd64: cmd/openshift cmd/oc pkg/sdn/plugin/sdn-cni-plugin vendor/github.com/containernetworking/cni/plugins/ipam/host-local vendor/github.com/containernetworking/cni/plugins/main/loopback
**go build github.com/openshift/origin/cmd/oc: /home/travis/.gimme/versions/go1.7.linux.amd64/pkg/tool/linux_amd64/link: signal: killed**
[ERROR] PID 1767: hack/common.sh:165: `local -a binaries=("$@")` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/common.sh:165: `local -a binaries=("$@")`
[INFO] 		  2: hack/build-go.sh:36: os::build::build_binaries
[INFO]   Exiting with code 1.
make: *** [build] Error 1
```

**Comment by coreydaley** [comment]
https://travis-ci.org/openshift/origin/builds/227273072

**Comment by coreydaley** [comment]
https://travis-ci.org/openshift/origin/builds/227168574

**Comment by stevekuznetsov** [comment]
I think this is the Travis infrastructure killing it on a timer our on resource consumption. See also https://github.com/openshift/origin/pull/13922

**Comment by jupierce** [comment]
@stevekuznetsov Looks like #13922 at least makes this not a P1 (fixes break with a speed penalty) and #13981 should eliminate that penalty. Agree?

**Comment by stevekuznetsov** [comment]
https://github.com/openshift/origin/pull/13981 will close this out, yes
